app/models/manageiq/providers/openstack/base_metrics_capture.rb

```diff
@@ -420,7 +420,7 @@ def parse_datetime(datetime)
   end
 
   def metric_service_from_settings
-    Settings[:workers][:worker_base][:queue_worker_base][:ems_metrics_collector_worker][:ems_metrics_openstack_default_service]
+    Settings.workers.ems_metrics_collector_worker.ems_metrics_openstack_default_service
```

Comment:
@agrare I don't understand how this could have blown up previously.  Like why is this rescued with StandardError?

Reply:
:shrug: looks like it was added here https://github.com/ManageIQ/manageiq/pull/13918

Looks like he was worried about the settings not being there?

Reply:
Strangely, the code doesn't care if the key is there but the value is nil, even though the log message talks about a nil value 🤷‍♂️

I can tweak this in a follow up to make more sense.